File: Backend/Lulus.ViewModels/Users/UpdateInfoRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lulus.ViewModels.Users
{
    public class UpdateInfoRequest
    {
        public string Token { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Password { get; set; }
        public string NewPassword { get; set; }
    }
}

File: Frontend/revolve/src/app/model/myaccount/updateInfoRequest.ts
export class UpdateInfoRequest{
    public token: String;
    public username:String;
    public email:String;
    public phone:String;
    public password:String;
    public newPassword:String;

    constructor(token:String, username:String, email:String, phone:String, password:String, newPass:String){
        this.token = token;
        this.username = username;
        this.email = email;
        this.phone = phone;
        this.password = password;
        this.newPassword = newPass;
    }
}